README.md

```diff
@@ -237,6 +237,23 @@ A simple but complete example to deploy a full monitoring stack on Kubernetes ca
 
 A complete example monitoring stack implemented in Docker Compose which can be found in the [examples](examples/config/docker) directory. 
 
+#### TLS HTTPS Support
+
+An usage example of how to deploy the exporter with TLS.
+
+Deployment:
+```shell
+$ pure-fa-openmetrics-exporter/out/bin/pure-fa-om-exporter -c cert.crt -k cert.key
+2023/08/01 12:00:00 Start Pure FlashArray exporter v1.0.9 on 0.0.0.0:9490
+```
+Testing:
+```shell
+$ cURL --header 'Authorization: Bearer xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' --request GET 'https://pure-fa-exporter.your.domain:9490/metrics/array?endpoint=arrayname.your.domain' --insecure --silent | grep purefa_info
```

Comment:
We need to standardize the variable we use for `<exporter-host>` . There's 3 different uses of that var in this document that use a different name

Reply:
I agree, I think an overhaul of this page is required now that it has grown. I propose we tackle these issues as part of a refresh and agree on the standardised names which is out of scope of this PR.